internal/providers/pluginfw/converters/converters_test.go

```diff
@@ -28,6 +28,8 @@ type DummyTfSdk struct {
 	EnumField         types.String                `tfsdk:"enum_field" tf:"optional"`
 	AdditionalField   types.String                `tfsdk:"additional_field" tf:"optional"`
 	DistinctField     types.String                `tfsdk:"distinct_field" tf:"optional"`
+	SliceStruct       []DummyNestedTfSdk          `tfsdk:"slice_struct" tf:"optional"`
+	SliceStructPtr    *[]DummyNestedTfSdk         `tfsdk:"slice_struct_ptr" tf:"optional"`
```

Comment:
Let's see if we can remove all pointers now. Optionality is expressed as nil/zero-length slice. Ideally, we don't need to use pointers anywhere or distinguish between pointers and concrete types in tfsdk structs.

Reply:
Removed all of those